### DISCUSSION TITLE
Newbie List of style shortcuts

### DISCUSSION THREAD

**Discussion by profhuster** [twbs/bootstrap#37418] [General]
TLDR: Need list of styling shortcuts

I am running Bootstrap on a Raspberry Pi Pico W (no Javascript) and I got my web site working. Cool. I followed Kevin McAleer's  video [here](https://www.youtube.com/watch?v=0sPPxIq4hg8).
He did a lot of formatting by saying and typing things like this is a nav link, give it some space around it that is what `ms-2` means. While going over the code
`<a class="navbar-brand mx-2" href="/">`

Another example he says I'll give a color theme `bg-light` This is all bootstrap. Make it rounded. ... so on.
`<div class="container bg-light rounded m-3 p-3">`

My question: How can I find a list of the styling shortcuts? These things like `mx-2`, `m-3`, `bg-light`, etc?

I looked through the documentation like for buttons, and on the page is CSS, but I haven't been able to translate the CSS into these styling shortcuts.

Please help.

**Comment by nkdas91** [comment]
Hi @profhuster 

Those classes can be found in the Utilities section from the Sidebar. 
Example - Spacing Utilities: https://getbootstrap.com/docs/5.2/utilities/spacing/

![image](https://user-images.githubusercontent.com/25012345/199411574-9363d635-c808-4fb6-a046-f7c453d8cc9b.png)

**Reply by profhuster** [reply]
Thanks! Now that you showed me where it is I can see `rounded`, `shadow`, `border`, etc.

**Reply by nkdas91** [reply]
Rounded: https://getbootstrap.com/docs/5.2/utilities/borders/#radius
Shadows: https://getbootstrap.com/docs/5.2/utilities/shadows/
Borders: https://getbootstrap.com/docs/5.2/utilities/borders/

For more, please check utilities from the Sidebar. You may also search for the required classes using the search box on the top navigation bar.
